### ISSUE TITLE
Add a keybind for quick discard

### ISSUE THREAD

**Issue by Jocelyn-JE** [cybensis/SPT-VR#6]
Maybe add a key combination to quickly discard an item, it would be better for looting

**Comment by cybensis** [comment]
That sounds like a good idea for sure! Do you have any suggestions for keybinds? I was thinking holding right grip then press B when hovering over an item

**Comment by Jocelyn-JE** [comment]
Yeah since we already have A + Right Grip and A + Left Grip for equip and quick move B + Right grip could be discard, maybe B + Left Grip could be a quick inspect?

**Comment by cybensis** [comment]
Sounds like a good idea to me! I did see someone else asking for a bind for quickly inspecting something so I should be able to add both those no problem
